woocommerce/woocommerce
# Product Collection: fix assymmetrical padding in Editor

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Original issue described asymmetrical padding on Product Collection in post editor. I realized the `editor.scss` file in Product Template was never imported. In addition, I had to increase the specificity of styles so they got applied.

Closes https://github.com/woocommerce/woocommerce/issues/44255



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create new post
2. Add Product Collection block and choose any collection
3. Focus on a block
4. **Expected:** there's no left and right margins next to product grid

Before | After
-- | --
See Green area on the left: <img width="670" alt="image" src="https://github.com/woocommerce/woocommerce/assets/20098064/b03480d0-b926-42e1-987b-415a0753ad05"> |  No margin/padding on left and right: <img width="674" alt="image" src="https://github.com/woocommerce/woocommerce/assets/20098064/e67a64e9-e40b-4209-b75b-991331393bc5">





### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
